ome_zarr/reader.py

```diff
@@ -405,57 +405,64 @@ def __init__(self, node: Node) -> None:
         LOGGER.info("well_data: %s", self.well_data)
 
         image_paths = [image["path"] for image in self.well_data.get("images")]
+
+        # Construct a 2D almost-square grid
         field_count = len(image_paths)
-        column_count = math.ceil(math.sqrt(field_count))
-        row_count = math.ceil(field_count / column_count)
+        self.row_count = math.ceil(math.sqrt(field_count))
+        self.column_count = math.ceil(field_count / self.row_count)
```

Comment:
Was there a reason to switch the order of column and row calculation here? Does it make any difference?

Reply:
This was not supposed to slip into the PR, indeed.
I just took it out in https://github.com/ome/ome-zarr-py/pull/209/commits/c965afe669543415c6ad1d97239d0d04dbde8863.

FYI, The reason was that for a non-square well I was getting an 8x9 shape instead of the 9x8 shape I needed. But I'm not expecting the `Well` class to support non-square geometry in a reliable way, in its current implementation. If needed, I would rather make the required shape more explicit (e.g. by passing an optional `grid_shape` argument to `Well`).